Gopkg.lock

```diff
@@ -528,6 +528,7 @@
     "k8s.io/apimachinery/pkg/api/errors",
     "k8s.io/apimachinery/pkg/apis/meta/v1",
     "k8s.io/apimachinery/pkg/util/sets",
+    "k8s.io/apimachinery/pkg/util/validation",
```

Comment:
is it used?

Reply:
I actually don't think it is. I could remove it if you like. I think it would just get overwritten anyway.